resources/SettingsTabComponents/SettingsPane.qml

```diff
@@ -71,7 +71,7 @@ Rectangle {
             columns: 7
             rows: 12
             rowSpacing: 1
-            height: parent.parent.height + Constants.settingsTab.paneScrollBufferHeight
+            Layout.preferredHeight: parent.height + Constants.settingsTab.paneScrollBufferHeight
```

Comment:
these fixes look like unrelated bugfixes... just confirming

Reply:
ah yea. will revert this height change.

will keep the parents -> parent changes since typo (reference error)